pom.xml

```diff
@@ -453,9 +453,9 @@
       </build>
     </profile>
     <profile>
-      <id>hadoop-2.7.2</id>
+      <id>hadoop-2.2.0</id>
```

Comment:
you can add a profile for hadoop-2.2.0, don't need to overwrite hadoop-2.7.2.  by default, should use hadoop-2.7.2